README.md

```diff
@@ -15,12 +15,18 @@ For further details see `requirements.txt`.
 * For example:
     * Pandas
 
+## Installation
```

Comment:
The `pandas` library is optional and only used in examples.  A nice way to include this into `setup.py` is to use `extras_require` argument for `setup` function. See https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html for example. 

Users can then install it using `pip install fastener[examples]`.  It would be nice to showcase and explain this in Installation part.